bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -344,19 +347,49 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
                 case CHANNEL_SENSOR_SLEEPTIME:
                     logger.debug("{}: Set sensor sleep time to {}", thingName, command);
-                    int value = ((DecimalType) command).intValue();
+                    int value = (int) getNumber(command);
                     value = value > 0 ? Math.max(SHELLY_MOTION_SLEEPTIME_OFFSET, value - SHELLY_MOTION_SLEEPTIME_OFFSET)
                             : 0;
                     api.setSleepTime(value);
                     break;
+                case CHANNEL_CONTROL_PROFILE:
+                    logger.debug("{}: Select profile {}", thingName, command);
+                    int profile = (int) getNumber(command);
+                    if (profile < 0 || profile > 5) {
+                        logger.info("{}: Invalid profile Id {} requested", thingName, profile);
```

Comment:
See above. Logging to Info. Please check all.

Reply:
Doesn't it make use to make the user aware of a mis-configuration. Otherwise nothing happens and there is no indication in the log with default config. How to find this out without a log message?

Reply:
If this is supposed to warn the user something is wrong, this should be WARN.

Reply:
ok, I reviewed and adjusted all log levels in this module